src/adcm_client/base.py

```diff
@@ -109,10 +104,6 @@ class ADCMApiError(Exception):
     pass
 
 
-class ResponseTooLong(Exception):
```

Comment:
ResponseTooLong and ActionHasIssues are needed exceptions and it is used on the current tests

Reply:
They didn't go anywhere, they just moved to base.py

Reply:
it breaks a backward compatibility with ADCM and bundle tests

Reply:
fixed